app/models/foreman_docker/service/actions/container/pull.rb

```diff
@@ -0,0 +1,28 @@
+module ForemanDocker
+  module Service
+    module Actions
+      module Container
+        class Pull < ::Actions::EntryAction
+          def plan(container)
+            action_subject(container)
+            plan_self(:container => container.id)
+          end
+
+          def run
+            container = ::Container.find(input[:container])
+            container.compute_resource.create_image(:fromImage => container.repository_pull_url)
```

Comment:
@pitr-ch Ok, so as I understand this, I should use `suspend` to offload the thread pool, then my action would run in a separate thread by using `Thread.new` and upon completion, `def run` will receive a `:done` event. 

Questions:
* I don't want to do anything in particular when it's done other than letting `Actions::Container::Start` run. I guess I can skip matching for `:done` and it'll be done automatically?
* Is `event` always going to be whatever I put in `suspended_action`? Or is there any list of possible events I should be careful with? 
* The first time `run` is called, after the `plan` phase, what's the value of `event`? nil?